packages/block-editor/src/components/font-family/index.js

```diff
@@ -51,14 +53,14 @@ export default function FontFamilyControl( {
 	}
 
 	return (
-		<SelectControl
+		<CustomSelectControl
 			__next40pxDefaultSize={ __next40pxDefaultSize }
 			__nextHasNoMarginBottom={ __nextHasNoMarginBottom }
 			label={ __( 'Font' ) }
-			options={ options }
 			value={ value }
 			onChange={ onChange }
```

Comment:
This will yield an error right now. We need to set the right value, something like:

```suggestion
			onChange={ ( { selectedItem } ) => onChange( selectedItem.style.fontFamily ) }
```

Reply:
Thanks for pointing this, fixed here: https://github.com/WordPress/gutenberg/pull/67118/commits/c39b0f1051a88f8298fba281e1d5877246973904